pkg/apis/core/v1/validation/validation.go

```diff
@@ -107,7 +107,8 @@ func ValidateNonnegativeQuantity(value resource.Quantity, fldPath *field.Path) f
 }
 
 // Validate compute resource typename.
-// Refer to docs/design/resources.md for more details.
+// Refer to https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md
```

Comment:
this file doesn't appear to match content about a quota resource name

Reply:
Nope, they are (almost) same file.
https://github.com/kubernetes/kubernetes/blob/bff3cb4df68d699285285c09c1c46be69ef1465a/docs/design/resources.md
vs
https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md

Resource type name [def here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#resource-types).